g2core/error.h

```diff
@@ -416,7 +415,7 @@ static const char stat_34[] = "Persistence error";
 static const char stat_35[] = "Bad status report setting";
 static const char stat_36[] = "Failed to get planner buffer";
 
-static const char stat_37[] = "Backplan hit running buffer";
+static const char stat_37[] = "37";
```

Comment:
This seems to be removing a text string, replacing it with "37".

Doesn't seem correct?

Reply:
> I removed the error message for code 37 since it appears to currently be marked as a placeholder.

TBH, I don't remember exactly why I believed it to be a placeholder, but I clearly thought about it at the time making it a separate commit and mentioning it in the PR message as well. If it's not correct however, then it shouldn't be included.  Either way is fine with me.